pil-analyzer/src/pil_analyzer.rs

```diff
@@ -257,6 +258,23 @@ impl PILAnalyzer {
         }
     }
 
+    pub fn struct_fields_check(&self) -> Result<(), Vec<Error>> {
+        let structs_exprs = self.all_children().filter_map(|expr| {
```

Comment:
I'm wondering if it would be better to do that inside `check_structs_fields`?

Reply:
Not sure, but the idea of having to pass `PILAnalyzer` (to make it possible to do `self.all_children()`), doesn't really convince me.